src/main/webapp/definitions/alfresco-content-models.yaml

```diff
@@ -0,0 +1,738 @@
+      - skipCount
+      - maxItems
+    properties:
+      count:
+        type: integer
+        format: int64
+        description: |
+          The number of objects in the entries array.
+      hasMoreItems:
+        type: boolean
+        description: |
+          A boolean value which is **true** if there are more entities in the collection
+          beyond those in this response. A true value means a request with a larger value
+          for the **skipCount** or the **maxItems** parameter will return more entities.
+      totalItems:
+        type: integer
+        format: int64
+        description: |
+          An integer describing the total number of entities in the collection.
+          The API might not be able to determine this value,
+          in which case this property will not be present.
+      skipCount:
+        type: integer
+        format: int64
+        description: |
+          An integer describing how many entities exist in the collection before
+          those included in this list. If there was no **skipCount** parameter then the 
+          default value is 0.
+      maxItems:
+        type: integer
+        format: int64
+        description: |
+          The value of the **maxItems** parameter used to generate this list.
+          If there was no **maxItems** parameter then the default value is 100.
+  QNamePaging:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/QNameEntry'
+          source:
+            $ref: '#/definitions/QName'
+  QNameEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/QName'
+  QName:
+    type: object
+    required:
+      - namespacePrefix
+      - namespaceUri
+      - name
+    properties:
+      namespacePrefix:
+        type: string
+      namespaceUri:
+        type: string
+      name:
+        type: string
+  ModelEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Model'
+  Model:
+    allOf:
+      - $ref: '#/definitions/QName'
+      - properties:
+          version:
+            description: the model version
+            type: string
+          author:
+            description: author of the model
+            type: string
+          description:
+            description: the model description
+            type: string
+  TypeEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Type'
+  Type:
+    allOf:
+      - $ref: '#/definitions/QName'
+      - properties:
+          title:
+            description: the human-readable type title
+            type: string
+          parent:
+            description: parent type defination
+            $ref: '#/definitions/Type'
```

Comment:
We probably want to use parentId here instead otherwise we'll have to expand the type hierarchy in the response which would get very expensive if you have a deep model structure. 

The client can make an extra call if they need parent type information.